test/cluster/test_system_clients_scan_perf.py

```
@@ -0,0 +1,159 @@
+    msg = (f"{label}: {NUM_CONNECTIONS} connections, smp={NUM_SHARDS}: {values}, "
+           f"RSS before/after/peak (kB): {rss_before}/{rss_after}/{rss_peak}, "
+           f"allocated before/after/peak (kB): {alloc_before}/{alloc_after}/{alloc_peak}")
+    logger.info(msg)
+    print(msg)
+    # xdist swallows stdout/logging; drop a plain file so results survive test.py's capture.
+    with open(RESULT_FILE, "a") as f:
+        f.write(msg + "\n")
+
+
+@pytest.mark.parametrize("concurrency", [1, NUM_CONCURRENT_SCANS])
+async def test_system_clients_scan_perf(request, manager: ScyllaClusterManager, concurrency: int):
```

Ruff (0.16.3)

[warning] Unused function argument: `request`

(ARG001)